packages/framework-provider-kubernetes/src/library/searcher-adapter.ts

```diff
@@ -0,0 +1,79 @@
+import { RedisAdapter } from './../services/redis-adapter'
+import {
+  BoosterConfig,
+  Logger,
+  InvalidParameterError,
+  FilterFor,
+  Operation,
+  ReadModelEnvelope,
+} from '@boostercloud/framework-types'
+
+export async function searchReadModel(
+  redis: RedisAdapter,
+  config: BoosterConfig,
+  logger: Logger,
+  readModelName: string,
+  filters: FilterFor<unknown>
+): Promise<Array<any>> {
+  logger.debug('Running search with the following filters: \n', filters)
+
+  const keys = await redis.keys(`rm_${readModelName}_*`, logger)
```

Comment:
rm_${readModelName}_* is not going to retrieve any results, since the separator now is _____ . I'll encourage you to use this syntax to generate the keys search term
`redis.keys(['rm', readModelName, '*'].join(RedisAdapter.keySeparator)`

Reply:
Done, and works like a charm 👌🏻